kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py

```diff
@@ -149,6 +151,12 @@ def __init__(self, name, init_config, instances):
 
                 instance['health_url'] = url
 
+            slis_instance = self.create_sli_prometheus_instance(instances[0])
+            self.slis_scraper_config = self.get_scraper_config(slis_instance)
+            self.detect_sli_endpoint(
+                self.get_http_handler(self.slis_scraper_config), slis_instance.get('prometheus_url')
+            )
```

Comment:
I'm wondering if we should the `sli_prometheus_instance` in the for loop ` for instance in instances:`, having on SLI prometheus instance for each instance.

I suppose we can do something like `instance['sli_prometheus'] = self.create_sli_prometheus_instance(instances[0])`

```suggestion
            slis_instance = self.create_sli_prometheus_instance(instances[0])
            instance['sli_scraper_config'] = self.get_scraper_config(slis_instance)
            self.detect_sli_endpoint(
                self.get_http_handler(self.slis_scraper_config), slis_instance.get('prometheus_url')
            )
```